### ISSUE TITLE
Slug string from URI-unsafe characters

### ISSUE THREAD

**Issue by kogai** [netlify/netlify-cms#501]
**- Do you want to request a *feature* or report a *bug*?**

FEATURE REQUEST
(Maybe, related to https://github.com/netlify/netlify-cms/pull/414)

**- What is the current behavior?**
In case of using `editorial workflow` and `GitHub backend`.

When we create entry with URI-unsafe characters in title like `日本語のタイトル` or `عنوان فارسی`, we can't create PR correctly (It is created with the title of empty string)

**- What is the expected behavior?**

If we want to create entry with URI-unsafe characters, we only do that slugify URI-safe characters.
As below

```
slugFn("日本語のタイトル") // expect "日本語のタイトル"
slugFn("日本語 Abc タイトル") // expect "日本語-abc-タイトル"
```

**- Please mention your node.js, and operating system version.**

`v8.2.1`

**Comment by erquhart** [comment]
Should have been addressed by #414, please re-open if still an issue.

**Comment by tech4him1** [comment]
#414 added the `slug` library, which accounts for 30.5% of our current bundle size. This should probably be changed to something a little lighter weight.

@biilmann also suggested using https://github.com/netlify/netlify-cms/blob/master/src/lib/urlHelper.js#L15
